docs/en/latest/plugins/fault-injection.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: fault-injection
+keywords:
+  - APISIX
+  - Plugin
+  - Fault Injection
+  - fault-injection
+description: This document contains information about the Apache APISIX fault-injection Plugin.
```

Comment:
Maybe License Checker failed due to markdown meta changes?

Reply:
Only supports the `title` attribute? I'm not sure, need to verify it.

Reply:
The other attributes were working when I tested. i.e, Keywords and descriptions were showing properly for the page.

Reply:
Maybe we need to make some changes in the configuration: https://github.com/apache/skywalking-eyes#configurations

I could not find any but maybe that could be the issue.

Reply:
> The other attributes were working when I tested. i.e, Keywords and descriptions were showing properly for the page.

Hi, Navendu, I'm not sure if it's because CI will skip `title` only. 

need @spacewander's help to check why CI will fail 🙏

Reply:
I am not familiar with the apache/skywalking-eyes.
Maybe you can open an issue there?
It is expected to work as we don't change the license header.